### PR TITLE
Updated descriptions of CR parameters

### PR DESCRIPTION
**Pull request by nitishkumar4**

### CODE REVIEW

File: operator/api/v1/csiscaleoperator_types.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type CSIScaleOperatorSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Attacher Node Selector",xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"
 	AttacherNodeSelector []CSINodeSelector `json:"attacherNodeSelector,omitempty"`
 
-	// clusters is a collection of GPFS CSICluster properties for the CSI driver to mount.
+	// clusters is a collection of spectrum scale cluster properties for the CSI driver to mount.
```
Comment (madhuthorat) [inline operator/api/v1/csiscaleoperator_types.go]: Spectrum Scale is name of a product, so please have first letter in capital in both the words "spectrum scale".
Same comment would apply to other changes using "spectrum scale"

Comment (amdabhad) [inline operator/api/v1/csiscaleoperator_types.go]: Let's have initials in caps like `Spectrum Scale` ? as it is a product name.

File: operator/api/v1/csiscaleoperator_types.go [hunk 2]
```diff
@@ -315,7 +315,7 @@ type CSICluster struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Secrets",xDescriptors="urn:alm:descriptor:io.kubernetes:Secret"
 	Secrets string `json:"secrets"` // TODO: Secrets should be Singular
 
-	// secureSslMode specifies if a secure SSL connection to connect to GPFS is required.
+	// secureSslMode specifies if a secure SSL connection to connect to spectrum scale is required.
```
Comment (madhuthorat) [inline operator/api/v1/csiscaleoperator_types.go]: "to spectrum scale" replace with "to Spectrum Scale cluster"

File: operator/api/v1/csiscaleoperator_types.go [hunk 3]
```diff
@@ -338,18 +338,18 @@ type CSIFilesystem struct {
...
-	// Remote CSICluster ID
+	// Remote spectrum scale cluster ID
 	RemoteCluster string `json:"remoteCluster,omitempty"`
 }
 
 // Defines the fields for REST API server information.
 type RestApi struct {
 
-	// The hostname of the REST server.
+	// guiHost is the hostname/IP of the spectrum scale GUI.
```
Comment (madhuthorat) [inline operator/api/v1/csiscaleoperator_types.go]: Reword as "guiHost specified the hostname/IP of the spectrum scale GUI node."

File: operator/api/v1/csiscaleoperator_types.go [hunk 4]
```diff
@@ -338,18 +338,18 @@ type CSIFilesystem struct {
...
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="GUI Host",xDescriptors="urn:alm:descriptor:com.tectonic.ui:label"
 	GuiHost string `json:"guiHost"`
 
-	// The port number running the REST server.
+	// guiPort is the port number of the spectrum scale GUI.
```
Comment (madhuthorat) [inline operator/api/v1/csiscaleoperator_types.go]: add word "node" at the end.

File: operator/api/v1/csiscaleoperator_types.go (hunk 1 above)
Comment (nitishkumar4) [reply operator/api/v1/csiscaleoperator_types.go]: Fixed.

Comment (nitishkumar4) [reply operator/api/v1/csiscaleoperator_types.go]: Fixed.

File: operator/api/v1/csiscaleoperator_types.go (hunk 2 above)
Comment (nitishkumar4) [reply operator/api/v1/csiscaleoperator_types.go]: Fixed.

File: operator/api/v1/csiscaleoperator_types.go (hunk 3 above)
Comment (nitishkumar4) [reply operator/api/v1/csiscaleoperator_types.go]: Fixed.

File: operator/api/v1/csiscaleoperator_types.go (hunk 4 above)
Comment (nitishkumar4) [reply operator/api/v1/csiscaleoperator_types.go]: Fixed.

**Review by madhuthorat — APPROVED** [review]

**Pull request merged**